classes/db/DbPDO.php

```diff
@@ -72,6 +72,8 @@ protected static function getPDO($host, $user, $password, $dbname, $timeout = 5)
                 PDO::MYSQL_ATTR_USE_BUFFERED_QUERY => true,
                 PDO::MYSQL_ATTR_INIT_COMMAND => 'SET NAMES utf8mb4',
                 PDO::MYSQL_ATTR_MULTI_STATEMENTS => _PS_ALLOW_MULTI_STATEMENTS_QUERIES_,
+                // FIX This option keeps all data as strings and stops automatic casting to integers, floats and other types
+                PDO::ATTR_STRINGIFY_FETCHES => true
```

Comment:
```suggestion
                PDO::ATTR_STRINGIFY_FETCHES => true,
```
You need to add this coma to respect coding style (or the CI will remain blocked)